src/onegov/api/models.py

```diff
@@ -206,3 +221,47 @@ def collection(self):
         """
 
         raise NotImplementedError()
+
+
+class AuthEndpoint:
+
+    def __init__(self, app):
+        self.app = app
+
+
+class ApiKey(Base):
+
+    __tablename__ = 'api_keys'
+
+    id = Column(
```

Comment:
I removed them because I got a mypy error I was not able to solve / understand: (Maybe you know it)

`src/onegov/api/models.py:245: error: Argument 1 to "Column" has incompatible type "type[onegov.core.orm.types.uuid_type.UUID]"; expected "type[TypeEngine[uuid.UUID]]"  [arg-type]`

this happens when I add

```python
    id: 'Column[uuid.UUID]' = Column(
        UUID,
        nullable=False,
        primary_key=True,
        default=uuid4
    )

    # the user that created the api key
    user_id: 'Column[uuid.UUID]' = Column(
        UUID, ForeignKey('users.id'), nullable=False
    ```

Reply:
Yes, it's because the two types don't map cleanly to each other in the sqlalchemy plugin, you can just ignore the type error for now, there's another UUID column further below which has the appropriate `type:ignore` comment.

In SQLAlchemy 2.0 we will be able to clean this up.